.tool-versions

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2023. University of Texas at Austin. All rights reserved.
```

Comment:
Please use a copy-left license for our code.

Reply:
I do not know what that means

Reply:
instead of MIT please use GPLv3

Reply:
`SPDX-License-Identifier: GPL-3.0-only`

Reply:
Are we being struct about this -- as in we should add another pre-commit like in the AMD repos?  Or should we just remove the stray licenses?

Reply:
If strict can you send the exact license lines you want please?

Reply:
That is the exact line I want to change. The other is fine.

Reply:
jk chris want it to be 
`SPDX-License-Identifier: BSD-2-Clause`